ceph_deploy/hosts/centos/install.py

```diff
@@ -4,6 +4,23 @@
 from ceph_deploy.util.paths import gpg
 
 
+def map_components(components):
```

Comment:
alright, I follow...  So the only thing that would be distro specific would be set of components that map to 'ceph' instead of their original name...

That makes sense.  :)  I'll give it a shot.